luma/core/sprite_system.py

```diff
@@ -0,0 +1,180 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2014-17 Richard Hull and contributors
+# See LICENSE.rst for details.
+
+import time
+from PIL import Image
+
```

Comment:
can this module get a docstring? I'd also use 2017 only in the copyright string to remind it's only available recently.